app/js/lib/foxbox/foxbox.js

```diff
@@ -466,4 +468,29 @@ export default class Foxbox extends Service {
         });
       });
   }
+
+  /**
```

Comment:
Trying to not spend too much time to make api as generic as possible as it's going to change again soon.